### PR TITLE
Update students.yml

### PR DESCRIPTION
**Pull request by lavavi**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

**Comment by lavavi** [comment]
lavavi

File: _data/students.yml [hunk 1]
```diff
@@ -140,3 +140,10 @@
   facebook: PL0YP1T1
   lat: 13.756331
   lng: 100.501765
+
+- name: Ashish Kumar Singh
+  country: India
+  github: lavavi
+  image: lavavi.jpg
```
Comment (lavavi) [inline _data/students.yml:147]: lavavi

**Review by anshumanv — CHANGES_REQUESTED** [review]
Please add your image in this directory - `img/students`

**Comment by YashKumarVerma** [comment]
@lavavi  please add the changes in the PR description too.

**Review by isuruAb — CHANGES_REQUESTED** [review]
hi @lavavi , please add a 240x240 sized image

**Comment by ParthS007** [comment]
@lavavi Please update your PR

**Pull request closed without merging**